mkdocs.yml

```diff
@@ -140,7 +134,6 @@ nav:
   - 'Script Extensions': 'research/script-extensions.md'
   - 'Signature Script Isolation and Fraud Proofs': 'research/signature-script-isolation-and-fraud-proofs.md'
   - 'Block Production Times': 'research/block-production-times.md'
-  - 'Transaction Extensions': 'research/transaction-extensions.md'
```

Comment:
No redirect for this

Reply:
Added redirect, though not sure if working...